graphstorm-processing/tests/test_dist_numerical_transformation.py

```diff
@@ -377,10 +427,11 @@ def test_rank_gauss(spark: SparkSession, check_df_schema, epsilon):
 
 @pytest.mark.parametrize("epsilon", [0.0, 1e-6])
 def test_rank_gauss_reshuffling(spark: SparkSession, check_df_schema, epsilon):
+    """Test rank-Gauss transformation results order with enforced shuffling."""
     # Create DF with 10k values
     random_values = np.random.rand(10**3, 1)
 
-    # Convert the array of values into a list of single-value lists
+    # Convert the array of values into a list of single-value tuples
```

Comment:
Is this a new change in this PR or we fixed it in the previous PR?

Reply:
We don't make any changes in the code, just making the comment more accurate.

```python
data = [(float(value),) for value in random_values]
```

`(float(value),)` is a single-element tuple (therefore immutable), not a list.